packages/ui-extensions/src/surfaces/checkout/components/Sheet/Sheet.doc.ts

```diff
@@ -5,7 +5,7 @@ import {getExample} from '../../helper.docs';
 const data: ReferenceEntityTemplateSchema = {
   name: 'Sheet',
   description:
-    'Sheet is designed to be used on top of other elements in a user interface and is typically bound to the bottom of a page. Sheet can contain and display various types of content such as forms, or informational messages.\n\nThe library automatically applies the [WAI-ARIA Dialog pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/) to both the activator and the sheet content.',
+    'The Sheet component displays essential information for customers at the bottom of the screen, appearing above other elements. Use it sparingly to avoid distracting customers during checkout. This component requires access to [Customer Privacy API](/docs/api/checkout-ui-extensions/unstable/configuration#collect-buyer-consent) to be rendered. \n\nThe library automatically applies the [WAI-ARIA Dialog pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/) to both the activator and the sheet content.',
```

Comment:
Just want confirm. Does this mean that you can technically use the Sheet component for any purpose, but it is designed for collecting buyer consent and requires it? It it doesn't have it, it won't render?